### PR TITLE
Fix types for static mode (#139)

### PR DESCRIPTION
**Pull request by 4a57**
PR for issue #139

### CODE REVIEW

**Comment by mcollina** [comment]
Can you please add a unit test for these new types?

**Comment by 4a57** [comment]
done

File: index.d.ts [hunk 1]
```diff
@@ -5,20 +5,19 @@ import * as SwaggerSchema from 'swagger-schema-official';
...
     routePrefix?: string;
     /**
-     *  To expose the documentation api
+     * To expose the documentation api
      * @default false
      */
     exposeRoute?: boolean;
+  }
+
+  interface FastifyDynamicSwaggerOptions extends FastifySwaggerOptions {
```
Comment (mcollina) [inline index.d.ts]: I think `mode?: 'dynamic';` should be kept here. Why did you remove it?

Comment (4a57) [reply index.d.ts]: Oh, right... Fixed
